guides/source/testing.md

```diff
@@ -535,7 +535,7 @@ class ActiveSupport::TestCase
 end
 ```
 
-Rails applications generated from JRuby will automatically include the `with: :threads` option.
+Rails applications generated from Ruby implementations that do not support fork will automatically include the `with: :threads` option.
```

Comment:
I think we should give concrete examples here, even if the actual check no longer sniffs the platform - most readers probably have no idea whether their Ruby implementation supports fork or not.

Let's go back to what you had before:

```suggestion
Rails applications generated from JRuby or TruffleRuby will automatically include the `with: :threads` option.
 ```

Reply:
@eugeneius I've updated back to the concrete example.